ui/pages/onboarding-flow/onboarding-flow.js

```diff
@@ -49,19 +50,31 @@ import MetaMetricsComponent from './metametrics/metametrics';
 export default function OnboardingFlow() {
   const [secretRecoveryPhrase, setSecretRecoveryPhrase] = useState('');
   const dispatch = useDispatch();
-  const currentLocation = useLocation();
+  const { pathName, search } = useLocation();
   const history = useHistory();
   const t = useI18nContext();
   const completedOnboarding = useSelector(getCompletedOnboarding);
   const seedPhraseBackedUp = useSelector(getSeedPhraseBackedUp);
   const nextRoute = useSelector(getFirstTimeFlowTypeRoute);
-
+  const isFromReminder = new URLSearchParams(search).get('isFromReminder');
   useEffect(() => {
-    if (completedOnboarding && seedPhraseBackedUp) {
+    if (completedOnboarding && !isFromReminder) {
       history.push(DEFAULT_ROUTE);
     }
   }, [history, completedOnboarding, seedPhraseBackedUp]);
```

Comment:
I believe we can update the dependencies here as well with the new changes

Reply:
done [here](https://github.com/MetaMask/metamask-extension/pull/16916/commits/134f878548f688b439f5da53f5a959d9ad499e24)